controllers/productController.js

```diff
@@ -0,0 +1,67 @@
+const productService = require('../services/productService');
+const { catchAsync } = require('../utils/error');
+
+const getProductDistanceByUser = catchAsync(async (req, res) => {
+  const { latitude, longitude, distanceMeter, optionArray, limit, offset } = req.query;
+  const userType = req.userType;
+  let sortBy = req.query.sortBy;
```

Comment:
어제 말씀드린 부분입니다!
요청의 query parameter에서 받아오는 값들을 다음과 같이 작성해주면 if 조건문을 사용하지 않아되 되기 때문에 좀 더 간결해질 것 같네요!

해당 부분 수정해주세요~!

```suggestion
  let sortBy = req.query.sortBy ? req.query.sortBy : "default";
```

Reply:
네 수정하겠습니다